storage_service/locations/models/pipeline_local.py

```diff
@@ -136,3 +136,17 @@ def move_from_storage_service(self, source_path, destination_path, package=None)
 
         # Move file
         return self.space.move_rsync(source_path, destination_path, assume_rsync_daemon=self.assume_rsync_daemon, rsync_password=self.rsync_password)
+
+    def isfile(self, path_):
+        """Verify that something is a file in this Space's context."""
+        LOGGER.info("Testing if isfile in pipeline file system: %s", path_)
+        basename = os.path.basename(path_)
+        if not len(basename):
+            return False
+        path_ = os.path.join(os.path.dirname(path_), "")
```

Comment:
Is this really necessary? Could `os.path.dirname(path_)` be enough?

Reply:
It's also used in the `browse` method like that, so I'd use that method and live it be.